.semgrepignore

```diff
@@ -0,0 +1,2 @@
+# Test data
```

Comment:
Why do we need this suddently?

Reply:
This was needed because we added a rule to check for `scheme: basic` in our test files. That caused the semgrep task to fail. I can remove it for the context of this PR given we aren't adding new rules